homeassistant/components/zwave_js/climate.py

```diff
@@ -275,6 +288,40 @@ def preset_modes(self) -> Optional[List[str]]:
         """Return a list of available preset modes."""
         return list(self._hvac_presets)
 
+    @property
+    def fan_mode(self) -> Optional[str]:
+        """Return the fan setting."""
+        if (
+            self._fan_mode
+            and self._fan_mode.value is not None
+            and str(self._fan_mode.value) in self._fan_mode.metadata.states
+        ):
+            return cast(str, self._fan_mode.metadata.states[str(self._fan_mode.value)])
```

Comment:
Do we still need to cast to string? We have typed the values to strings:
https://github.com/home-assistant-libs/zwave-js-server-python/blob/749f4e69728bfc25d3000387ffc01982492d5942/zwave_js_server/model/value.py#L24

Reply:
It seems so:
`homeassistant/components/zwave_js/climate.py:299: error: Returning Any from function declared to return "Optional[str]"  [no-any-return]
Found 1 error in 1 file (checked 1 source file)`

Reply:
Yes, I see it. We haven't typed the property detailed enough:
https://github.com/home-assistant-libs/zwave-js-server-python/blob/749f4e69728bfc25d3000387ffc01982492d5942/zwave_js_server/model/value.py#L124

Ok. That's something for a future PR to address.